apps/frontend/src/libs/components/task-card/task-card.tsx

```diff
@@ -134,8 +147,8 @@ const TaskCard: React.FC<Properties> = ({
 			{isNoteOpen && (
 				<Notes
 					onNoteClose={handleNoteClose}
-					onSubmit={onAddTaskNote}
-					task={task}
+					onSubmit={onAddTaskNote as (payload: TaskNoteRequestDto) => void}
```

Comment:
In Properties type you have the same function type. You can move it to type AddTaskNotehandler, reuse it in Properties and here